src/test/java/GmailServletTest.java

```diff
@@ -12,25 +12,18 @@
 // See the License for the specific language governing permissions and
```

Comment:
Please see my comment regarding using DI to access `GmailResponseUtility`. That should greatly simplify these tests.

Reply:
Done